### PR TITLE
Reminder: Fix reminder time being in past

### PR DESCRIPTION
**Pull request by ArcasCZ**
Fixes situation in which member set reminder into past.

### CODE REVIEW

**Review by sinus-x — APPROVED** [review]
ACK

**Comment by sinus-x** [comment]
There seem to be merge issues which I cannot resolve here over GitHub UI. This can be merged once they are fixed.

File: reminder/module.py [hunk 1]
```diff
@@ -252,6 +261,15 @@ async def remind(
             )
             return
 
+        if date < datetime.now():
+            await ctx.reply(
+                _(
+                    ctx,
+                    "Time already passed.",
```
Comment (sinus-x) [inline reminder/module.py:268]: Actually, this message already exists in 'reminder reschedule' as

> Reschedule time must be in furuter.

Please, use

> Time must be in future.

for all of the messages.

**Review by sinus-x — CHANGES_REQUESTED** [review]

File: reminder/module.py (hunk 1 above)
Comment (ArcasCZ) [reply reminder/module.py]: OK, done.

**Comment by sinus-x** [comment]
There are still merge issues here.

**Review by sinus-x — APPROVED** [review]
ACK

**Pull request merged**